src/components/input/input.tsx

```diff
@@ -63,37 +63,47 @@ export class Input implements LabelableComponent, FormComponent, InteractiveComp
   //
   //--------------------------------------------------------------------------
 
-  /** specify the alignment of the value of the input */
+  /** Specifies the text alignment of the component's value. */
   @Prop({ reflect: true }) alignment: Position = "start";
 
-  /** should the input autofocus */
+  /**
+   * When true, the component is focused on page load.
+   *
+   * @mdn [autofocus](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/autofocus)
+   */
   @Prop() autofocus = false;
 
   /**
-   * optionally display a clear button that displays when field has a value
-   * shows by default for search, time, date
-   * will not display for type="textarea"
+   * When true, a clear button is displayed when the component has a value. The clear button shows by default for "search", "time", and "date" types, and will not display for the "textarea" type.
    */
   @Prop({ reflect: true }) clearable = false;
 
-  /** is the input disabled  */
+  /**
+   * When true, interaction is prevented and the component is displayed with lower opacity.
+   *
+   * @mdn [disabled](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/disabled)
+   */
   @Prop({ reflect: true }) disabled = false;
 
   @Watch("disabled")
   disabledWatcher(): void {
     this.setDisabledAction();
   }
 
-  /** for number values, displays the locale's group separator */
+  /**
+   * When true, number values are displayed with the locale's group separator.
+   */
   @Prop() groupSeparator = false;
 
-  /** when true, the component will not be visible */
+  /**
+   * when true, the component will not be visible.
```

Comment:
Should this be ~when~ **When** true,...? Wondering since this is the "hidden" prop, if we should use language indicating it will hide the component as opposed to it not being visible? (e.g., When true, the component will be hidden and won't be rendered in the browser)?

Reply:
I try not to use the word I'm defining in the description. Also setting the `hidden` flag doesn't remove the element from the DOM, so it is technically still rendered. It just isn't visible.